WireAnalytics/Sources/WireAnalytics/AnalyticsSession.swift

```diff
@@ -44,3 +104,10 @@ public struct AnalyticsSession: AnalyticsSessionProtocol {
         countly.recordEvent(event.rawValue)
     }
 }
+
+// Extension to logRound
+extension Int {
+    func logRound() -> Int {
+        return Int(log2(Double(self)).rounded())
+    }
+}
```

Comment:
I suggest to move to a new file, as well as document what it does (see the Confluence docs about why we have this)

Reply:
Done: https://github.com/wireapp/wire-ios/commit/60e1c1a08b6be26eff43996ec29220f8489c3829